poll_suggestion/models.py

```diff
@@ -0,0 +1,10 @@
+from django.db import models
+
+
+class poll_suggestion(models.Model):
+    date = models.CharField(max_length=40)
+    hour = models.CharField(max_length=20)
+    user_name = models.CharField(max_length=20)
+
+    def __str__(self):
+        return self.date,self.hour
```

Comment:
Please add a whitespace like this to pass the flake8 test:
```suggestion
        return self.date, self.hour
```